uint8 incident_type
uint8 repeat
